src/transformers/models/idefics/modeling_idefics.py

```diff
@@ -875,24 +875,27 @@ def forward(
         attention_mask: Optional[torch.Tensor] = None,
         image_hidden_states: Optional[torch.Tensor] = None,
         image_attention_mask: Optional[torch.Tensor] = None,
+        cross_attention_gate: Optional[torch.Tensor] = None,
         output_attentions: Optional[bool] = False,
         use_cache: Optional[bool] = False,
         past_key_value: Optional[Tuple[torch.Tensor]] = None,
-        no_images: Optional[bool] = False,
     ) -> Tuple[torch.FloatTensor, Optional[Tuple[torch.FloatTensor, torch.FloatTensor]]]:
```

Comment:
As discussed in private, there is no need for a deprecation cycle since this is not an argument provided to idefics_model, but just only to idefics cross_attention